sklearn/manifold/t_sne.py

```diff
@@ -619,22 +631,22 @@ class TSNE(BaseEstimator):
         Journal of Machine Learning Research 15(Oct):3221-3245, 2014.
         https://lvdmaaten.github.io/publications/papers/JMLR_2014.pdf
     """
-    # Control the number of exploration iterations with early_exaggeration on
-    _EXPLORATION_N_ITER = 250
 
     # Control the number of iterations between progress checks
     _N_ITER_CHECK = 50
 
     def __init__(self, n_components=2, perplexity=30.0,
                  early_exaggeration=12.0, learning_rate=200.0, n_iter=1000,
-                 n_iter_without_progress=300, min_grad_norm=1e-7,
-                 metric="euclidean", init="random", verbose=0,
-                 random_state=None, method='barnes_hut', angle=0.5):
+                 n_iter_early_exag=250, n_iter_without_progress=300,
```

Comment:
For backward compatibility, please place, `n_iter_early_exag` at the end of the function signature.